src/hog.jl

```diff
@@ -0,0 +1,99 @@
+"""
+```
+hog_params = HOG([orientations = 9], [cell_size = 8], [block_size = 2], [block_stride = 1], [norm_method = "L2-norm"])
+```
```

Comment:
Would be good to have a brief description of what this is for, before going into detail about the parameters.